modules/nw-egress-ips-about.adoc

```diff
@@ -20,6 +20,8 @@ To ensure that you can reliably allow access to the server from only that specif
 
 An egress IP address assigned to a namespace is different from an egress router, which is used to send traffic to specific destinations.
 
+Automatic attachment and detachment of egress IP addresses for traffic between nodes are available. This allows for traffic from many pods in namespaces to have a consistent source IP address to locations outside of the cluster. This also supports OpenShift SDN and OVN-Kubernetes as a default network provider.
+
```

Comment:
OpenShift SDN and OVN-Kubernetes, which is the default networking plug-in in Red Hat OpenShift Networking in {product-title} {product-version}.

Approved language per Marc Curry in regard to rebrand. Thanks!